client/src/components/ProductDetails/StyleSelector.jsx

```diff
@@ -1,41 +1,30 @@
+/* eslint-disable react/jsx-key */
 /* eslint-disable react/prop-types */
```

Comment:
If you want to avoid commenting out this line of code, you could import PropTypes from 'prop-types'; and then follow this [tutorial](https://reactjs.org/docs/typechecking-with-proptypes.html) to add a few lines of code to the bottom of the file